README.md

```diff
@@ -1,24 +1,44 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Set up a "Hello world" project with react-rails gem
 
-Things you may want to cover:
+> A simple application for study purposes. It combines rails as a back-end and react as a front-end library.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- React
 
-* Configuration
+## Getting Started
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+- Copy and clone this link locally `https://github.com/tugsem/my_new_rails_app.git`.
+- Go to the repository folder in your command prompt `cd my_new_rails_app`.
 
-* Database initialization
+- Follow these steps to run the app:
+  - run `bundle` and `yarn install` to install all dependencies and ruby gems
+  - run `rails db:create`,`rails db:migrate`,`rails db:seed` to create database
+  - run `rails s`
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## Authors
 
-* Deployment instructions
+👤 **Tuğsem Saranlı**
 
-* ...
+- GitHub: [@githubhandle](https://github.com/tugsem)
+- Twitter: [@twitterhandle](https://twitter.com/tugsemSaranli)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/tugsem)
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use your user_name for each social media to make the **`## Authors`**  section of your README more professional.
e.g.
```
- GitHub: [@tugsem](https://github.com/tugsem)
- ...
```